tools/gptfdisk/Makefile

```diff
@@ -0,0 +1,36 @@
+#
+# Copyright (C) 2015 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=gptfdisk
```

Comment:
Submit a separate pull request to add a tools that is not used (yet) ? 
I can make a separate commit only for the tools, but i don't think the PR will be merged if the tool is not used...